content/en/tutorials/s3-static-website-terraform/index.md

```diff
@@ -0,0 +1,336 @@
+---
+title: "Host a static website locally using Simple Storage Service (S3) and Terraform with LocalStack"
+linkTitle: "Host a static website locally using Simple Storage Service (S3) and Terraform with LocalStack"
+weight: 4
+description: >
+  Host a static website using a Simple Storage Service (S3) bucket to serve static content by provisioning the infrastructure using Terraform in LocalStack. Learn how to configure S3 buckets locally for testing and integration, and make use of LocalStack's S3 API & `tflocal` CLI to provision infrastructure locally.
+cascade:
+  type: docs
+---
+
+[AWS Simple Storage Service (S3)](https://aws.amazon.com/s3/) is a proprietary object storage solution that can store an unlimited number of objects for many use cases. S3 is a highly scalable, durable and reliable service that we can use for various use cases: hosting a static site, handling big data analytics, managing application logs, storing web assets and much more!
```

Comment:
No action necessary here, but it's usually a good idea to have one sentence per line in markdown. This helps with the diffs.